### PR TITLE
Call this.props.onChange only if it's defined

### PR DESCRIPTION
**Pull request by ar7n**
Prevents errors if you doesn't pass onChange to component

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ var MaskedInput = React.createClass({
         this._updateInputSelection()
       }
     }
-    if (this.props.onChange) {
+    if (typeof this.props.onChange !== 'undefined') {
```
Comment (jquense) [inline src/index.js]: `if (this.props.onChange)` is fine here. They provide the same level of safety here (and the rest of the cases below).

**Comment by iamdustan** [comment]
@ar7n if you could switch these all to `if (this.props.onChange)` like @jquense mentioned I’ll get this merged in and published. Thanks!

**Comment by ar7n** [comment]
@iamdustan i'm done

**Comment by iamdustan** [comment]
published as 3.1.3.

**Pull request merged**